demo-targets/bodgeit/README.md

```diff
@@ -52,32 +52,236 @@ The bodgeit chart can be deployed via helm:
 helm upgrade --install bodgeit secureCodeBox/bodgeit
 ```
 
-## Values
-
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| affinity | object | `{}` |  |
-| annotations | object | `{}` | add annotations to the deployment, service and pods |
-| fullnameOverride | string | `""` |  |
-| image.pullPolicy | string | `"IfNotPresent"` | Image pull policy. One of Always, Never, IfNotPresent. Defaults to Always if :latest tag is specified, or IfNotPresent otherwise. More info: https://kubernetes.io/docs/concepts/containers/images#updating-images |
-| image.repository | string | `"docker.io/securecodebox/bodgeit"` | Container Image containing the bodgeit |
-| image.tag | string | defaults to the "latest" version because the appVersion tag is not available at docker.io | The image tag |
-| imagePullSecrets | list | `[]` | Define imagePullSecrets when a private registry is used (see: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) |
-| ingress.annotations | object | `{}` |  |
-| ingress.enabled | bool | `false` |  |
-| ingress.hosts[0].host | string | `"chart-example.local"` |  |
-| ingress.hosts[0].paths | list | `[]` |  |
-| ingress.tls | list | `[]` |  |
-| labels | object | `{}` | add labels to the deployment, service and pods |
-| nameOverride | string | `""` |  |
-| nodeSelector | object | `{}` |  |
-| podSecurityContext | object | `{}` |  |
-| replicaCount | int | `1` |  |
-| resources | object | `{}` |  |
-| securityContext | object | `{}` |  |
-| service.port | int | `8080` |  |
-| service.type | string | `"ClusterIP"` |  |
-| tolerations | list | `[]` |  |
+<table>
```

Comment:
was this done by hand?
is this a change in helm-docs or required by docusaurus?

I'd prefer to keep a markdown table if possible.